packages/protocol/contracts/common/FeeBurner.sol

```diff
@@ -0,0 +1,425 @@
+pragma solidity ^0.5.13;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+
+import "./UsingRegistry.sol";
+import "../common/Freezable.sol";
+import "../common/FixidityLib.sol";
+import "../common/Initializable.sol";
+
+import "../stability/StableToken.sol"; // TODO check if this can be interface
+import "../common/interfaces/ICeloVersionedContract.sol";
+import "../common/interfaces/ICeloToken.sol";
+import "../stability/interfaces/IExchange.sol";
+import "../stability/interfaces/ISortedOracles.sol";
+
+// Using the minimal required signatures in the interfaces so more contracts could be compatible
+import "../uniswap/interfaces/IUniswapV2RouterMin.sol";
+import "../uniswap/interfaces/IUniswapV2FactoryMin.sol";
+import "../uniswap/interfaces/IUniswapV2PairMin.sol";
+
+contract FeeBurner is Ownable, Initializable, UsingRegistry, ICeloVersionedContract, Freezable {
```

Comment:
We should add getVersionNumber to this contract

Reply:
It's a little below